aict_tools/configuration.py

```diff
@@ -142,6 +129,35 @@ def __init__(self, config):
         if 'separator' in config:
             self.separator = SeparatorConfig(config)
 
+    def parse_cta(self, config):
+        SIMPLE_OPTIONS = set([
+            'size',
+            'energy_unit',
+            'events_key'
+        ])
+        if SIMPLE_OPTIONS.intersection(config.keys()):
+            raise TypeError(
+                'You are trying to use configuration keys for the simple '
+                'data format while setting the data_format to "CTA". '
+                'There should be no reason to do this as the CTA DL1-format is fixed'
+            )
+        self.telescopes = config.get('telescopes', None)
+        self.true_energy_column = config.get('true_energy_column')
```

Comment:
We also don't need this, right? This will always be `simulation/event/subarray/shower.true_energy` or at least be data model dependent.